include/core/Array.hpp

```diff
@@ -57,6 +57,11 @@ class Object;
 class Array {
 	godot_array _godot_array;
 
+	friend class Variant;
+	inline Array(const godot_array other) {
```

Comment:
Two little things:
* You might want to make the constructor `explicit`
* Parameter should probably be a const-reference (even if `sizeof(godot_array)` is small _in current implementation_)

Same for the `Dictionary` constructor.

Reply:
Done, thanks again for the improvements.